math/src/field/traits.rs

```diff
@@ -230,6 +230,9 @@ pub trait FieldElement:
 /// the modulus of the field should be a prime number of the form `k` * 2^`n` + 1 (a Proth prime),
 /// where `n` is relatively large (e.g., greater than 32).
 pub trait StarkField: FieldElement<BaseField = Self> {
+    // CONSTANTS
+    // =============================================================================================
```

Comment:
minor nit: we usually use `----` for internal section separators instead of `=====` (also applicable to other changes in this file).

Reply:
Ah yes - fixed